src/Racer.ts

```diff
@@ -0,0 +1,25 @@
+import { EventEmitter } from 'events';
+import { Model, RootModel } from './Model';
+import * as util from './util';
+
+export class Racer extends EventEmitter {
```

Comment:
- does this need to even be a class? (derby inherits)
- does this even use `EventEmitter`? (derby may)